packages/api/src/command/medical/ihe-result/create-document-query-result.ts

```diff
@@ -0,0 +1,15 @@
+import { DocumentQueryResponseIncoming } from "@metriport/ihe-gateway-sdk";
+import { DocumentQueryResultModel } from "../../../models/medical/document-query-result";
+import { DefaultPayload } from "./shared";
+
+export async function createDocumentQueryResult(
+  defaultPayload: DefaultPayload,
+  status: string,
+  response: DocumentQueryResponseIncoming
```

Comment:
Why not having all of them in a single type? Like, from the caller's perspective, it's odd to have a param called `defaultPayload` and part of the info sent there and the rest as their own, separated parameter.

ditto for others